packages/react/src/components/DotcomShell/README.md

```diff
@@ -0,0 +1,83 @@
+# DotcomShell
+
+> The DotcomShell component includes the `Masthead`, `Footer`, and `Content`
+> components, all wrapped in a UI shell using Carbon's grid.
+
+## Getting started
+
+The `DotcomShell` uses Carbon's `Content` component. All user-provided content
```

Comment:
`Carbon's Content component` seems a little off. Users can put in pretty much anything in the `DotcomShell`, right? We should just say that any content can be placed in the `DotcomShell`, from HTML to other React components.

Reply:
I see what you're saying. I specifically mentioned the `Content` component so they'd know where to look for info if they needed